jaeger_client/local_agent_net.py

```diff
@@ -51,15 +62,19 @@ class LocalAgentSender(TBufferedTransport):
     end of the batch span submission call.
     """
 
-    def __init__(self, host, sampling_port, reporting_port, io_loop=None):
+    def __init__(self, host, sampling_port, reporting_port, io_loop=None, throttling_port=None):
         # IOLoop
         self._thread_loop = None
         self.io_loop = io_loop or self._create_new_thread_loop()
 
-        # http sampling
-        self.local_agent_http = LocalAgentHTTP(host, sampling_port)
+        # HTTP sampling
+        self.sampling_http = LocalAgentHTTP(host, sampling_port)
```

Comment:
this is technically still a public field, let's keep the old name for now

Reply:
Sure. With Python I find it difficult to distinguish public and prive.